web/src/lib/domain-logic/isLicenseDataFromDatabaseDataMoreRecent.test.ts

```diff
@@ -0,0 +1,80 @@
+import { isLicenseDataFromDatabaseDataMoreRecent } from "@/lib/domain-logic/isLicenseDataFromDatabaseDataMoreRecent";
+import { generateBusiness, generateLicenseData, getCurrentDate } from "@businessnjgovnavigator/shared/";
+
+const currentDate = getCurrentDate();
+const currentDateISOString = currentDate.toISOString();
+const currentDatePlusOneHourISOString = currentDate.add(1, "hour").toISOString();
+
+describe("isLicenseDataFromDatabaseDataMoreRecent", () => {
+  it("returns true when license data is undefined in businessFromDb but defined in businessFromUpdateQueue", () => {
+    const result = isLicenseDataFromDatabaseDataMoreRecent({
+      businessFromDb: generateBusiness({ licenseData: undefined }),
+      businessFromUpdateQueue: generateBusiness({}),
+    });
+
+    expect(result).toBeTruthy();
+  });
+
+  it("returns true when licence data from businessFromUpdateQueue is not defined but licence data from businessFromDb is", () => {
```

Comment:
```suggestion
  it("returns true when license data from businessFromUpdateQueue is not defined but license data from businessFromDb is", () => {
```
[dust] Typos. Also in other tests.